### PR TITLE
Rewrite Discs

### PR DESCRIPTION
**Pull request by Yesterday17**

### CODE REVIEW

File: album/KSLA-0067.toml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+[[discs.tracks]]
+title = "Philosophyz Game size Ver."
+
+[[discs.tracks]]
+title = "闇の彼方へ Game size Ver."
+
+[[discs.tracks]]
+title = "Philosophyz off vocal Ver."
+artist = "折戸伸治、a2c(MintJam)"
```
Comment (tuzi3040) [inline album/KSLA-0067.toml:27]: Composer or arranger or both?

File: album/KSLA-0076.toml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+type = "normal"
+
+[[discs.tracks]]
+title = "過日憐歌"
+artist = "mao"
+type = "normal"
+
+[[discs.tracks]]
+title = "大地～Radiance"
+artist = "坂本英城"
```
Comment (tuzi3040) [inline album/KSLA-0076.toml:42]: > 7. Ground - Radiance
>   Composer: Soshi Hosoi
>   Arranger: Hideki Sakamoto (noisycroak Co.,Ltd.)
> 
>   1st violin: Yui
>   2nd violin: maiko
>   Viola: Namiko
>   Cello: Mariko
>   Recording: Yoshiyuki Watanabe (VICTOR STUDIO)
>  Mixing: 河野真人

Should be `細井聡司`. Seems all artists of absolute track need a refetch.

File: album/KSLA-0091.toml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+[[discs.tracks]]
+title = "Harvest"
+artist = "多田葵"
+type = "normal"
+
+[[discs.tracks]]
+title = "Potted one"
+
+[[discs.tracks]]
+title = "刈り"
```
Comment (tuzi3040) [inline album/KSLA-0091.toml:22]: Remix of original `刈り`, original composer is `井内舞子`.
Artist of remixed absolute tracks should also be same as original tracks.
Seems adding optional (or required depend on track type) field `composer` and `arranger` is worth considering.

File: album/VA-00112D.toml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+[album]
```
Comment (tuzi3040) [inline album/VA-00112D.toml:1]: Animation Blue-Ray of `Rewrite` have like 6 or 7 disks of CD `月刊テラ・風祭学院支局 特別編` included.
Maybe you want to collect them too. :) But hard and expensive :(

**Review by tuzi3040 — CHANGES_REQUESTED** [review]

File: album/VA-00112D.toml (hunk 4 above)
Comment (Yesterday17) [reply album/VA-00112D.toml:1]: image

Radio CDs enclosed with Blu-ray(`ANZX-xxx`) can be easily found. The hard points are the other ones. Just skip them :)

File: album/KSLA-0067.toml (hunk 1 above)
Comment (Yesterday17) [reply album/KSLA-0067.toml:28]: Both

File: album/KSLA-0076.toml (hunk 2 above)
Comment (Yesterday17) [reply album/KSLA-0076.toml:43]: image
We may need to write both of them to the artist field.

File: album/KSLA-0067.toml (hunk 1 above)
Comment (tuzi3040) [reply album/KSLA-0067.toml:28]: Better to be standardized.
Also , I prefer `artist` to be composer, and `arranger` as an optional attribute when `type` is absolute or instrumental. If `type` is normal, `artist` should be performer, and `composer` `arranger` are optional.

File: album/KSLA-0118.toml [hunk 5]
```diff
@@ -0,0 +1,34 @@
+[album]
+album_id = "c71c0a96-3291-4359-b867-6e5b6aa33ddf"
+title = "Philosophyz／ささやかなはじまり"
+artist = "Key Sounds Label"
+date = 2016-07-27
+type = "normal"
+catalog = "KSLA-0118"
+tags = ["アニメ「Rewrite」"]
```
Comment (tuzi3040) [inline album/KSLA-0118.toml:8]:
```suggestion
tags = ["TVアニメ「Rewrite」"]
```

**Review by tuzi3040 — APPROVED** [review]

**Pull request merged**